package.json

```diff
@@ -9,6 +9,7 @@
     "build:development": "POET_ENV=development webpack",
     "build:testing": "POET_ENV=testing webpack",
     "build:preview": "stylelint './src/**/*.scss' && tslint -p ./tsconfig.json && npm run build:testing",
+    "build:staging": "POET_ENV=staging webpack --mode=production",
```

Comment:
I think you have to remove `--mode=production`, because explorer-web doesn't have the `webpack v4` yet

Reply:
I've removed that. But still failing